diff -aur ./stress-ng-0.20.00/Makefile ../build/extbld/third_party/cmd/stress_ng/stress-ng-0.20.00/Makefile
--- ./stress-ng-0.20.00/Makefile	2026-01-01 01:52:06.000000000 +0300
+++ ../build/extbld/third_party/cmd/stress_ng/stress-ng-0.20.00/Makefile	2026-02-13 15:15:56.867461994 +0300
@@ -138,7 +138,7 @@
 MACHINE := $(shell make -f Makefile.machine)
 ifneq ($(PRESERVE_CFLAGS),1)
 ifneq ($(MACHINE),$(filter $(MACHINE),alpha hppa ia64))
-flag = -Wformat -fstack-protector-strong -Werror=format-security
+flag = -Wformat -Werror=format-security
 #
 # add -D_FORTIFY_SOURCE=2 if _FORTIFY_SOURCE is not already defined
 #
@@ -264,7 +264,7 @@
 # Static flags, only to be used when using GCC
 #
 ifeq ($(STATIC),1)
-override LDFLAGS += -static -z muldefs -ffunction-sections -fdata-sections -Wl,--gc-sections
+override LDFLAGS += -static -ffunction-sections -fdata-sections
 override CFLAGS += -DBUILD_STATIC
 endif
 
@@ -561,7 +561,6 @@
 	stress-handle.c \
 	stress-hash.c \
 	stress-hdd.c \
-	stress-heapsort.c \
 	stress-hrtimers.c \
 	stress-hsearch.c \
 	stress-hyperbolic.c \
@@ -590,7 +589,6 @@
 	stress-landlock.c \
 	stress-lease.c \
 	stress-led.c \
-	stress-link.c \
 	stress-list.c \
 	stress-llc-affinity.c \
 	stress-loadavg.c \
@@ -602,7 +600,6 @@
 	stress-logmath.c \
 	stress-longjmp.c \
 	stress-loop.c \
-	stress-lsearch.c \
 	stress-lsm.c \
 	stress-madvise.c \
 	stress-malloc.c \
@@ -611,11 +608,9 @@
 	stress-mcontend.c \
 	stress-membarrier.c \
 	stress-memcpy.c \
-	stress-memfd.c \
 	stress-memhotplug.c \
 	stress-memrate.c \
 	stress-memthrash.c \
-	stress-mergesort.c \
 	stress-metamix.c \
 	stress-mincore.c \
 	stress-min-nanosleep.c \
@@ -630,7 +625,6 @@
 	stress-mmapfixed.c \
 	stress-mmapfork.c \
 	stress-mmaphuge.c \
-	stress-mmapmany.c \
 	stress-mmaprandom.c \
 	stress-mmaptorture.c \
 	stress-module.c \
@@ -642,7 +636,6 @@
 	stress-mseal.c \
 	stress-msg.c \
 	stress-msync.c \
-	stress-msyncmany.c \
 	stress-mtx.c \
 	stress-munmap.c \
 	stress-mutex.c \
@@ -686,7 +679,6 @@
 	stress-quota.c \
 	stress-qsort.c \
 	stress-race-sched.c \
-	stress-radixsort.c \
 	stress-randlist.c \
 	stress-ramfs.c \
 	stress-rawdev.c \
@@ -726,7 +718,6 @@
 	stress-shm-sysv.c \
 	stress-sigabrt.c \
 	stress-sigbus.c \
-	stress-sigchld.c \
 	stress-sigfd.c \
 	stress-sigfpe.c \
 	stress-sighup.c \
@@ -760,7 +751,6 @@
 	stress-spinmem.c \
 	stress-splice.c \
 	stress-stack.c \
-	stress-stackmmap.c \
 	stress-statmount.c \
 	stress-str.c \
 	stress-stream.c \
@@ -791,7 +781,6 @@
 	stress-umask.c \
 	stress-umount.c \
 	stress-unlink.c \
-	stress-unshare.c \
 	stress-uprobe.c \
 	stress-urandom.c \
 	stress-userfaultfd.c \
@@ -821,10 +810,6 @@
 	stress-x86cpuid.c \
 	stress-x86syscall.c \
 	stress-xattr.c \
-	stress-yield.c \
-	stress-zero.c \
-	stress-zlib.c \
-	stress-zombie.c \
 
 SRC = $(CORE_SRC) $(CORE_SRC_GEN) $(STRESS_SRC)
 OBJS += stress-eigen-ops.o
diff -aur ./stress-ng-0.20.00/core-stressors.h ../build/extbld/third_party/cmd/stress_ng/stress-ng-0.20.00/core-stressors.h
--- ./stress-ng-0.20.00/core-stressors.h	2026-01-01 01:52:06.000000000 +0300
+++ ../build/extbld/third_party/cmd/stress_ng/stress-ng-0.20.00/core-stressors.h	2026-02-13 15:29:52.667258113 +0300
@@ -130,7 +130,6 @@
 	MACRO(handle)		\
 	MACRO(hash)		\
 	MACRO(hdd)		\
-	MACRO(heapsort)		\
 	MACRO(hrtimers)		\
 	MACRO(hsearch)		\
 	MACRO(hyperbolic)	\
@@ -159,7 +158,6 @@
 	MACRO(landlock)		\
 	MACRO(lease)		\
 	MACRO(led)		\
-	MACRO(link)		\
 	MACRO(list)		\
 	MACRO(llc_affinity)	\
 	MACRO(loadavg)		\
@@ -171,7 +169,6 @@
 	MACRO(logmath)		\
 	MACRO(longjmp)		\
 	MACRO(loop)		\
-	MACRO(lsearch)		\
 	MACRO(lsm)		\
 	MACRO(madvise)		\
 	MACRO(malloc)		\
@@ -180,11 +177,9 @@
 	MACRO(mcontend)		\
 	MACRO(membarrier)	\
 	MACRO(memcpy)		\
-	MACRO(memfd)		\
 	MACRO(memhotplug)	\
 	MACRO(memrate)		\
 	MACRO(memthrash)	\
-	MACRO(mergesort)	\
 	MACRO(metamix)		\
 	MACRO(mincore)		\
 	MACRO(min_nanosleep)	\
@@ -199,7 +194,6 @@
 	MACRO(mmapfixed)	\
 	MACRO(mmapfork)		\
 	MACRO(mmaphuge)		\
-	MACRO(mmapmany)		\
 	MACRO(mmaprandom)	\
 	MACRO(mmaptorture)	\
 	MACRO(module)		\
@@ -211,7 +205,6 @@
 	MACRO(mseal)		\
 	MACRO(msg)		\
 	MACRO(msync)		\
-	MACRO(msyncmany)	\
 	MACRO(mtx)		\
 	MACRO(munmap)		\
 	MACRO(mutex)		\
@@ -255,7 +248,6 @@
 	MACRO(qsort)		\
 	MACRO(quota)		\
 	MACRO(race_sched)	\
-	MACRO(radixsort)	\
 	MACRO(ramfs)		\
 	MACRO(randlist)		\
 	MACRO(rawdev)		\
@@ -295,7 +287,6 @@
 	MACRO(shm_sysv)		\
 	MACRO(sigabrt)		\
 	MACRO(sigbus)		\
-	MACRO(sigchld)		\
 	MACRO(sigfd)		\
 	MACRO(sigfpe)		\
 	MACRO(sighup)		\
@@ -329,13 +320,11 @@
 	MACRO(spinmem)		\
 	MACRO(splice)		\
 	MACRO(stack)		\
-	MACRO(stackmmap)	\
 	MACRO(statmount)	\
 	MACRO(str)		\
 	MACRO(stream)		\
 	MACRO(swap)		\
 	MACRO(switch)		\
-	MACRO(symlink)		\
 	MACRO(sync_file)	\
 	MACRO(syncload)		\
 	MACRO(sysbadaddr)	\
@@ -361,7 +350,6 @@
 	MACRO(umask)		\
 	MACRO(umount)		\
 	MACRO(unlink)		\
-	MACRO(unshare)		\
 	MACRO(uprobe)		\
 	MACRO(urandom)		\
 	MACRO(userfaultfd)	\
@@ -391,11 +379,7 @@
 	MACRO(workload)		\
 	MACRO(x86cpuid)		\
 	MACRO(x86syscall)	\
-	MACRO(xattr)		\
-	MACRO(yield)		\
-	MACRO(zero)		\
-	MACRO(zlib)		\
-	MACRO(zombie)
+	MACRO(xattr)
 
 /*
  *  Declaration of stress_*_info object
